apps/datahub/src/app/home/search/record-preview-datahub/record-preview-datahub.component.css

```diff
@@ -0,0 +1,9 @@
+@media only screen and (min-width: 640px) {
+    gn-ui-favorite-star {
+        top: -1.125em;
+        right: 0.4em;
+    }
+    .icons {
+        right: 0.4em;
+    }
+}
```

Comment:
Why can't we use tailwind classes for these ?

Reply:
good point! it works with tw classes actually, I was sure it didn't!!